Makefile

```diff
@@ -92,9 +92,9 @@ docker-push: ## Push docker image with the manager.
 # To properly provided solutions that supports more than one platform you should use this option.
 PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
 .PHONY: docker-buildx
-docker-buildx: test ## Build and push docker image for the manager for cross-platform support
-	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
+docker-buildx: ## Build and push docker image for the manager for cross-platform support
+	# copy existing Dockerfile and insert --platform=${TARGETPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{TARGETPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{TARGETPLATFORM\}/' Dockerfile > Dockerfile.cross
```

Comment:
@jon-nfc It looks like this is the most up to date version of this command, I haven't tested this out, but I think we should probably take the latest guidance from the ansible-operator-sdk folks:
* https://github.com/operator-framework/ansible-operator-plugins/blob/bbe095f316eda31423b18b8d5e6bafddd524f558/testdata/memcached-molecule-operator/Makefile#L40-L55

If you agree, could you please update the PR to reflect what is shown there?

Reply:
see https://github.com/ansible/eda-server-operator/pull/158#issuecomment-1879993218.

response wouldn't post to this thread